js/src/viewer/manifestsPanel.js

```diff
@@ -129,15 +239,310 @@
            _this.hide();
         },
 
+        // Handler for when manifest data is loaded for the first time
```

Comment:
The side effect is adding a new list item to the manifest list UI. I remember this already being in the code prior to work on the collection tree, this is just added documentation.